plugins/generator-1.20.1/forge-1.20.1/procedures/entity_isaggressive.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity} instanceof Mob _mobEnt${cbi} ? _mobEnt${cbi}.isAggressive():false)
```

Comment:
Maybe this approach would be better:
`(${input$entity} instanceof Mob _mobEnt${cbi} && _mobEnt${cbi}.isAggressive())`

Reply:
@Galo223344, would you mind implementing this solution?